src/planwise/client/current_project/handlers.cljs

```diff
@@ -219,3 +262,101 @@
  in-current-project
  (fn [db [_ new-zoom]]
    (assoc-in db [:map-view :zoom] new-zoom)))
+
+;; ---------------------------------------------------------------------------
+;; Sharing handlers
+
+(register-handler
+ :current-project/open-share-dialog
+ in-current-project
+ (fn [db [_]]
+   ; TODO: Fire a request to update the project shares when opening the dialog to ensure it is up to date
+   ; What happens if the user deletes an item from the list before the request returns? In that case,
+   ; the item that was just deleted will re-appear.
```

Comment:
How about rendering the list with the invalid data, but allowing changes (ie revoking access) only after its state is `asdf/valid?` again?